Makefile.am

```diff
@@ -17,6 +17,7 @@ WIN32_BUILD_FILES = \
 EXTRA_DIST = \
 	autogen.sh \
 	scripts/gen-api-gtkdoc.py \
+        geany.appdata.xml.in \
```

Comment:
Sure, because  GitHub uses tab stops of 8 and doesn't display tabs or spaces themselves, so it alignes,but it should be exactly one tab like the other lines, not 4 spaces.  And you can see in the diff that it's not aligned.

```diff
diff --git a/Makefile.am b/Makefile.am
index c5d2a90..ffa9b33 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -17,7 +17,7 @@ WIN32_BUILD_FILES = \
 EXTRA_DIST = \
 	autogen.sh \
 	scripts/gen-api-gtkdoc.py \
-        geany.appdata.xml.in \
+	geany.appdata.xml.in \
 	geany.desktop.in \
 	geany.pc.in \
 	geany.spec \
@@ -87,8 +87,7 @@ desktop_in_files = geany.desktop
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
 
-appdata_in_files = geany.appdata.xml.in
-
 appdatadir = $(datadir)/appdata
+appdata_in_files = geany.appdata.xml.in
 nodist_appdata_DATA = $(appdata_in_files:.xml.in=.xml)
 @INTLTOOL_XML_RULE@
```